media_driver/linux/common/os/mos_context_specific.h

```diff
@@ -202,6 +202,19 @@ class OsContextSpecific : public OsContext
     void SetSliceCount(uint32_t *pSliceCount);
 
 #endif
+    //!
+    //! \brief  Get the context priority from KMD
+    //! \param  [in, out] pPriority
+    //!         Pointer to the priority of current gpu context.
+    //!
+    void GetGpuPriority(int32_t *pPriority);
```

Comment:
Which platform are these functions for?
Maybe we also need to add these functions in OsContextSpecificNext.

Reply:
It is a general feature. It is validated on Gen12 TGL.